eras/shelley/impl/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Version history for `cardano-ledger-shelley`
 
+## 1.5.0.0
```

Comment:
Removing an instance is a major version change, so this is correct.

But, it also needs a bumped in the `cardano-ledger-shelley.cabal` and bounds updated in all of the dependencies.